### ISSUE TITLE
Request: Add "ignore" userflair option

### ISSUE THREAD

**Issue by d-reddi** [PitchforkAssistant/devvit-flood-assistant#8]
Hi there!
Thank you for creating this devvit app, it's great!  Would it be possible to add a section for user flair ID(s) to be ignored when checking posts? I think it would be useful for instances where known or trusted users would post past the threshold. 

Thanks again!

**Comment by PitchforkAssistant** [comment]
That seems like a pretty simple addition. It would just be a check in the same place as the is moderator and is contributor checks.

**Comment by d-reddi** [comment]
Yes, that and somewhere to put the list of flair template ID(s) that it should ignore.  I don't think it should ignore all flairs, just the ones specified.
